### PR TITLE
Update the label when selection, path, or selection path value changes.

### PR DESCRIPTION
**Pull request by liuyang-li**
@azirbel @cyril-sf @Addepar/pretzel

### CODE REVIEW

File: src/select.coffee [hunk 1]
```diff
@@ -179,9 +179,15 @@ Ember.AddeparMixins.ResizeHandlerMixin,
...
   .property 'itemView'
 
-  selectedLabel: Ember.computed ->
-    Ember.get @get('selection'), @get('optionLabelPath')
-  .property 'selection', 'optionLabelPath'
+  init: ->
+    @_super()
+    @selectionLabelPathChanges()
+
+  selectionLabelPathChanges: Ember.observer ->
```
Comment (cyril-sf) [inline src/select.coffee]: `selectionLabelPathChanges: Ember.on 'init', Ember.observer ->` and you can delete `init`

**Comment by cyril-sf** [comment]
This looks good. Can you add a unit test?

**Comment by azirbel** [comment]
This is a great solution! Code LGTM, I'll second the idea of a unit test. Nitpick, but can you rename `selectionLabelPathChanges` to `selectionLabelPathObserver` (which is our conventional name for observers), or maybe `selectedLabelGenerator` (since this one is special)?

File: src/select.coffee (hunk 1 above)
Comment (liuyang-li) [inline src/select.coffee]: Done.

**Comment by liuyang-li** [comment]
A unit test has been added, and the observer was renamed.

**Pull request merged**